src/agent/driver-test-runner.js

```diff
@@ -208,6 +208,10 @@ export function validateKeysFromCommand(command) {
   for (let { id } of command.keypresses) {
     // PAGE_DOWN and PAGE_UP are the only commands that have the extra _ inside a key
     id = id.replace(/(PAGE)_(DOWN|UP)/, '$1$2');
+    // + is used to connect keys that are pressed simultaneously in v2 tests
+    id = id.replace('+', '_');
+    // `UP_ARROW`, `DOWN_ARROW`, etc are sent as `up`, `down`, etc
+    id = command.id.replace(/_ARROW/g, '');
```

Comment:
```suggestion
    id = id.replace(/_ARROW/g, '');
```

(Also, chaining the replacements would be a little more dry and consistent.)

Reply:
Yeah! wow! thanks for noticing!